deploy/iso/minikube-iso/configs/minikube_defconfig

```diff
@@ -21,28 +19,27 @@ BR2_ROOTFS_USERS_TABLES="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/use
 BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/rootfs-overlay"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_VERSION=y
-BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.9.64"
+BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.16.14"
 BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
-BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/linux-4.9_defconfig"
+BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/linux_defconfig"
+BR2_LINUX_KERNEL_NEEDS_HOST_LIBELF=y
 BR2_PACKAGE_GZIP=y
 BR2_PACKAGE_XZ=y
+BR2_PACKAGE_SYSDIG=y
 BR2_PACKAGE_FINDUTILS=y
 BR2_PACKAGE_GIT=y
-BR2_PACKAGE_JQ=y
 BR2_PACKAGE_CIFS_UTILS=y
 BR2_PACKAGE_E2TOOLS=y
 BR2_PACKAGE_NFS_UTILS=y
+BR2_PACKAGE_SSHFS=y
 BR2_PACKAGE_PARTED=y
 BR2_PACKAGE_CA_CERTIFICATES=y
-BR2_PACKAGE_LIBFUSE=y
-BR2_PACKAGE_LIBCURL=y
 BR2_PACKAGE_CURL=y
 BR2_PACKAGE_BRIDGE_UTILS=y
 BR2_PACKAGE_EBTABLES=y
 BR2_PACKAGE_ETHTOOL=y
 BR2_PACKAGE_IPROUTE2=y
 BR2_PACKAGE_IPTABLES=y
-BR2_PACKAGE_OPENSSH=y
```

Comment:
Why is this removed?

Reply:
It's still built. It's now selected by default.